### DISCUSSION TITLE
wrong alias when parent folder name equals library name

### DISCUSSION THREAD

**Discussion by wvq** [justkey007/tsc-alias#163] [General]
A simple example: 

src/fs/test.ts
```typescript
import { mkdir } from 'fs'
export default mkdir

```

compiled `tsc & tsc-alias`

dist/fs/test.js
```js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const fs_1 = require("../fs");
exports.default = fs_1.mkdir;
//# sourceMappingURL=test.js.map
```

`const fs_1 = require("../fs");`  should be `const fs_1 = require("fs");`

**Comment by raouldeheer** [comment]
@wvq Please read the first question from the FAQ https://github.com/justkey007/tsc-alias/discussions/110
